src/osx/cocoa/stdpaths.mm

```diff
@@ -139,4 +139,13 @@ ConfigFileConv WXUNUSED(conv)) const
     return fn.GetFullName();
 }
 
+wxString wxStandardPaths::GetSharedLibrariesDir() const
+{
+    // Shared libraries on OSX should be stored inside the
+    // <Bundle.app>/Contents/Frameworks
+    wxFileName fn( GetExecutablePath() );
+    wxString temp = fn.GetPath() + wxFileName::GetPathSeparator() + ".." + wxFileName::GetPathSeparator() + "Frameworks" + wxFileName::GetPathSeparator();
```

Comment:
This will have `..` in the result. There should be some way to resolve this path I assume.

Reply:
@MaartenBent ,
Fixed in the upcoming commit. Thx.